src/components/search/findly-sdk.js

```diff
@@ -6473,6 +6478,9 @@ FindlySDK.prototype.searchEventBinding = function (
                             var showAllHTML = _self.customTemplateObj.renderMessage(msgData);
                             if (!$("body").hasClass("top-down")) {
                               $(".search-body").empty().append(showAllHTML);
+                              if(_self.vars.isBuilder){
```

Comment:
@BharadwajAd-Kore : you can use isDev instead of isBuilder key , no need to create new keys
@SunilSi-kore